Objects/unicodeobject.c

```diff
@@ -514,7 +518,11 @@ _PyUnicode_CheckConsistency(PyObject *op, int check_content)
         else {
             PyUnicodeObject *unicode = (PyUnicodeObject *)op;
```

Comment:
I think we need to enclose as well this `unicode` variable as it will be unused if data is not used in the next line.

Reply:
Yes, now the compiler complains about `unicode`.